rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -543,52 +543,34 @@ error_code sceNpDrmIsAvailable2(vm::cptr<u8> k_licensee_addr, vm::cptr<char> drm
 	return npDrmIsAvailable(k_licensee_addr, drm_path);
 }
 
-error_code sceNpDrmVerifyUpgradeLicense(vm::cptr<char> content_id)
+error_code npDrmVerifyUpgradeLicense(vm::cptr<char> content_id)
 {
-	sceNp.warning("sceNpDrmVerifyUpgradeLicense(content_id=*0x%x)", content_id);
-
 	if (!content_id)
 	{
 		return SCE_NP_DRM_ERROR_INVALID_PARAM;
 	}
 
 	const std::string content_str(content_id.get_ptr(), std::find(content_id.get_ptr(), content_id.get_ptr() + 0x2f, '\0'));
```

Comment:
0x2f actually looks like a magic number in the code. Maybe you could make a named constant somewhere?

Reply:
Don't know, I've just kept this bit from the original function.

Reply:
I suppose it is a _content_str_ length, so it's less confusing. Not a big deal anyway.